docs/reference/filters.md

```diff
@@ -286,6 +286,28 @@ corsOrigin("https://www.example.org")
 corsOrigin("https://www.example.org", "http://localhost:9001")
 ```
 
+> [!NOTE]
```

Comment:
This likely renders differently on github and https://opensource.zalando.com/skipper/ so its better to avoid. 
IMO plain text will do just fine

Reply:
yes, exactly